components/message/Message.module.scss

```diff
@@ -4,20 +4,20 @@
   flex-direction: column;
```

Comment:
we're not heavily using this in saas yet but if we were this might have needd follow up work in saas